docs/operations/export_api.md

```diff
@@ -455,80 +504,22 @@ status:
 
-Next create a secret in the same namespace on the target cluster. Note: the token value will be a header that is passed to the server by CDI.
+Assuming there is a running VirtualMachineExport called `example-export` and the same namespace exists in the target cluster. The name of the kubeconfig of the target cluster is named `kubeconfig-target`, to clone the vm into the target cluster run the following commands:
 
-```yaml
-apiVersion: v1
-kind: Secret
-metadata:
-  name: secret-headers
-stringData:
-  token: "x-kubevirt-export-token:1234567890ab"
+```bash
+$ virtctl vmexport download example-export --manifest --include-secret --output=import.yaml
+$ kubectl apply -f import.yaml --kubeconfig=kubeconfig-target
 ```
-Note: make sure there is no ` ` between the `:` and the actual token, otherwise the space is sent as part of the header, the authentication will fail.
-Now we can go ahead and import the disk image into the target cluster using a data volume. For convenience, I put the data volume inside a data volume template section of the same VM spec as the source:
 
-```yaml
-apiVersion: kubevirt.io/v1
-kind: VirtualMachine
-metadata:
-  labels:
-    kubevirt.io/vm: vm-example-datavolume
-  name: example-target-vm
-spec:
-  dataVolumeTemplates:
-  - metadata:
-      creationTimestamp: null
-      name: example-dv
-    spec:
-      source:
-        http:
-          url: "https://virt-exportproxy-example.example.com/api/export.kubevirt.io/v1alpha1/namespaces/example/virtualmachineexports/example-export/volumes/example-dv/disk.img.gz"
-          certConfigMap: router-cert
-          secretExtraHeaders:
-          - secret-headers
-      storage:
-        accessModes:
-        - ReadWriteOnce
-        resources:
-          requests:
-            storage: 20Gi
-        storageClassName: target-local
-  running: false
-  template:
-    metadata:
-      labels:
-        kubevirt.io/vm: vm-example-datavolume
-    spec:
-      domain:
-        devices:
-          disks:
-          - disk:
-              bus: virtio
-            name: datavolumedisk1
-        resources:
-          requests:
-            memory: 2Gi
-      terminationGracePeriodSeconds: 0
-      volumes:
-      - dataVolume:
-          name: example-dv
-        name: datavolumedisk1
-```
+The first command generates the yaml and writes it to `import.yaml`. The second command applies the generated yaml to the target cluster. It is possible to combine the two commands writing to standard `out` with the first command, and piping it into the second command. Use this option if the export token should not be written to a file anywhere. This will create the VM in the target cluster, and provides CDI in the target cluster with everything required to import the disk images.
 
```

Comment:
There are more good examples in your PR, maybe we should add them?

Examples :
get manifest for existing export, that will NOT include header secret
```bash
$ virtctl vmexport download export --manifest
```
create vm export for vm example, and get the manifest, it will NOT include header secret
```bash
$ virtctl vmexport download export --manifest --vm=example
```
create vm export for vm snapshot example, and get the manifest, it will NOT include header secret
```bash
$ virtctl vmexport download export --manifest --snap=example
```
get manifest for existing export, that will include the header secret
```bash
$ virtctl vmexport download export --manifest --include-secret
```
get manifest for existing export, that will NOT include the header secret, in json format
```bash
$ virtctl vmexport download export --manifest --manifest-output-format=json
```
get manifest for existing export, that will include the header secret writing to a file
```bash
$ virtctl vmexport download export --manifest --include-secret --output=manifest.yaml
```

Reply:
The `virtctl export` should have examples of all these combinations. it would make sense to keep them there so if they change the documentation is not out of date.

Reply:
Ok, thanks